src/CMake/VisItFunctions.cmake

```diff
@@ -213,3 +216,38 @@ function(visit_install_mpich)
             DIRECTORY_PERMISSIONS ${vdp})
 endfunction()
 
+# need a function different than VISIT_INSTALL_TARGETS because we cannot use
+# the EXPORT unless all of the target's visit dependencies are also exported.
```

Comment:
Not sure it matters too awful much but I read "visit" not as the product name here at first.